src/main/java/com/budgeteers/financetracker/api/controllers/health/HealthController.java

```diff
@@ -0,0 +1,29 @@
+package com.budgeteers.financetracker.api.controllers.health;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/health")
+public class HealthController {
+
+    private static int counter = 0;
```

Comment:
This was for demonstration purposes. We can get rid of this counter logic in the next PR.